### ISSUE TITLE
Some issues 

### ISSUE THREAD

**Issue by DEVUVO** [mbruel/scenePacker#1]
Hello , i got some issues while using the cool app

1 - when i drop a few folder and each folder contains sub folder it does auto pack each sub folder separately

it should Not do that unless someone ask for it , if i want to pack a folder it should just do like winrar does with multi folder it packs the whole folder separately with anything inside it in a single rar or multi rar parts  ..

in this case no need to pack all dropped folder in one pack , if anyone wants this option they can just put all folders together in one folder and drop the main folder into the app

2 - to save HDD spaces and keep 1 copy , an option such as delete after pack would be very useful same as in winrar 
delete files after its done archiving

3 - a fixed comment = away to add a fixed comment or to import it from .nfo or .txt file would be excellent

that's all . everything else works perfect , 

can't wait to see the next version . great work as always.

**Comment by mbruel** [comment]
> 1 - when i drop a few folder and each folder contains sub folder it does auto pack each sub folder separately

yes it was the spec you gave me... the title of the app is "pack files/folders individually"
The goal is to automate packing thus doing it on all entries you give separately.
if you want to pack all the subfolders together, then you just put them in a single root folder and drop that root folder.



> 2 - to save HDD spaces and keep 1 copy , an option such as delete after pack would be very useful same as in winrar
> delete files after its done archiving

Could be added easily, I'll see when I've time. I'm taking some vacations those days, I'll see in October when I'll be back on the computer. Send me an email around 7th of October so I don't forget ;)



> 3 - a fixed comment = away to add a fixed comment or to import it from .nfo or .txt file would be excellent

fixed comment to go where? in a option of the rar comment?
I don't really see the sense if you pack several movies for example...

**Comment by DEVUVO** [comment]
> 1 - when i drop a few folder and each folder contains sub folder it does auto pack each sub folder separately

yes it was the spec you gave me... the title of the app is "pack files/folders individually"
The goal is to automate packing thus doing it on all entries you give separately.
if you want to pack all the subfolders together, then you just put them in a single root folder and drop that root folder.


================

This is maybe it was my fault explaining 

if there is one folder and the folder have files such as pdf files and then there is a sub folder contains a 1 extra file

should be all packed in one pack not each file into a single out put and then the sub folder in a single out put as well

instead if you got some time please fix this cause it does split everything from where it's belongs to

to be more clear

if files been add to the app then compress every file separate and if there is a folder dropped to the app put every folder and all files and subfolder inside this folder all in one rar set

thank you , 

about the comment , its for personal comment for each set to leave a note for my self when i backup yes i mean the rar comment , if not easy just skip it , i use comment in my rar compress from my default winrar profile n, maybe its not easy to code or takes time , thank you i will wait for you , have a great vacation.

**Comment by mbruel** [comment]
you can't add files to the app, only folders.
Those folders can include either files or subfolders. Each of them will be compressed (recursively for the subfolders)
So in your case you need to drop the parent folder of the one containing the pdf, then you'll get what you want.

**Comment by DEVUVO** [comment]
Alright , what if i have a parent folder that include many folder and i only want few folders to be compressed without touched the other folders ? 

in this case i have to copy like 500 folder or even more to another temp location 

i use everything search app to locate my work folders really quick and i can drop the folders into the app

if you can help make the scenepacker compress all dropped folder separately that would be great without compress the sub folder of each folder , 

and if the file support is not possible i will just use winrar right click instead , 

thank you i will donate :)
